File: AD-HOC/3048 - Secret Sequence/2374.java
import java.util.Scanner;

class Main {

    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);
        int sequence = 0, aux = 0;
        int N = scan.nextInt();
        for(int i=0; i<N; i++){
            int V = scan.nextInt();
            if(V!=aux){
                aux=V;
                sequence++;
            }
        }
        System.out.println(sequence);
        scan.close();
    }
}

File: AD-HOC/3048 - Secret Sequence/2374.c
#include <stdio.h>
 
int main() {
 
    int N, V,sequence = 0, aux = 0;

    scanf("%i", &N);
    for(int i=0; i<N; i++){
        scanf("%i", &V);
        if(V!=aux){
            aux = V;
            sequence++;
        }
    }
    
    printf("%i\n", sequence);
    return 0;
}